### PR TITLE
wavesj improvements

### PR DESCRIPTION
**Pull request by DrBlast**
added blockheaders, reserved balance and fixed orderstatusinfo

### CODE REVIEW

File: src/main/java/com/wavesplatform/wavesj/Node.java [hunk 1]
```diff
@@ -299,6 +306,25 @@ public OrderStatusInfo getOrderStatus(String orderId, AssetPair assetPair) throw
         return parse(r, ORDER_LIST);
     }
 
+    public Map<String, Long> getTradableBalance(AssetPair pair, String address) throws IOException {
+        String path = String.format("/matcher/orderbook/%s/%s/tradableBalance/%s", pair.getAmountAsset(), pair.getPriceAsset(), address);
+        HttpResponse r = exec(request(path));
+        return parse(r, RESERVED);
+    }
+
+    private Map<String, Long> getReserved(PrivateKeyAccount account, String path) throws IOException {
```
Comment (petermz) [inline src/main/java/com/wavesplatform/wavesj/Node.java]: Why do you need this function? It's only used in one place

File: pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.wavesplatform</groupId>
     <artifactId>wavesj</artifactId>
-    <version>0.10</version>
+    <version>0.10.1</version>
```
[the remark this answers is not in the document]
Comment (DrBlast) [reply pom.xml]: done

File: src/main/java/com/wavesplatform/wavesj/BlockHeaders.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class represents a block.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BlockHeaders {
```
[the remark this answers is not in the document]
Comment (DrBlast) [reply src/main/java/com/wavesplatform/wavesj/BlockHeaders.java]: ok

File: src/main/java/com/wavesplatform/wavesj/Node.java (hunk 1 above)
Comment (DrBlast) [reply src/main/java/com/wavesplatform/wavesj/Node.java]: yes, it's useless. done

**Pull request merged**